validation-test/stdlib/Prototypes/PersistentVector.swift.gyb

```diff
@@ -1676,13 +1676,12 @@ func == <Key, Value> (
   return lhs.taggedPointer == rhs.taggedPointer
 }
 
-final class _NativePVDictionaryStorageRef<Key : Hashable, Value>
-  : NonObjectiveCBase {
+final class _NativePVDictionaryStorageRef<Key : Hashable, Value> {
```

Comment:
While you are modifying this code, could you add an explicit `internal` modifier?